### PR TITLE
Add renderer property according to Highchart fonctionnality

### PR DESCRIPTION
**Pull request by kevinLB**
Hello @avdaredevil,
One more thing!
I added "renderer" property according to specs : http://api.highcharts.com/highcharts/Renderer

It allow us to manipulate chart and add some stuff like that:
this.$.myChart.renderer.image('https://www.maxprog.com/img/cat.jpg', 34.5, 50.5, 50, 50).add();

There "renderer" property is the same as the javascript object.

Thanks! :)

### CODE REVIEW

File: highcharts-behavior.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
             yZoom: {type: Boolean, value: false, reflectToAttribute: true, observer: "_zoomUpdate"},
             loading: {type: Boolean, value: false, observer: "_loadingUpdate", reflectToAttribute:true},
             loadingMessage: {type: String, value: ""},
+            renderer: {type: Object, computed: "_renderer(_chart)"},
```
Comment (avdaredevil) [inline highcharts-behavior.html]: You might just be able to do:
```javascript
renderer: {type: Object, computed: "get(_chart.renderer)"},
```

This way, as the renderer changes (but not the `_chart` object, ex. during initialization), the renderer is automatically binded, without supporting functions.

Comment (kevinLB) [inline highcharts-behavior.html]: I just tried this update but it throw an error: Cannot read property 'image' of undefined.
And yes I didn't notice the mutation during the init phase so, may I suggest this update instead ?

`renderer: {type: Object, computed: "_renderer(_chart.renderer)"},`

Then the mutation will be triggered only when "_chart.renderer" change (according to Polymer specs for mutation) : https://www.polymer-project.org/1.0/docs/devguide/observers#define-a-computed-property

Seems good ?

**Comment by kevinLB** [comment]
Usually we use "_" to say "not public" so, but OK there is no private state/functions.
It's just to be as close as we could from the main library!
This feature is not "worth" but I needed to access to renderer object and I think it's convenient to expose it as public.
The object exists for Stocks and Maps too so we have to keep the feature on baseBehavior for clarity.

Thanks @avdaredevil

**Comment by avdaredevil** [comment]
Ok let's ship this feature as well!

File: highcharts-behavior.html (hunk 1 above)
Comment (avdaredevil) [inline highcharts-behavior.html]: I don't see why the `get` wouldn't work. The `'image' of undefined` does not seem to be an error on the Highcharts side, but rather your code accessing the render object before the component has initialized.

Could you confirm @kevinLB

Comment (avdaredevil) [inline highcharts-behavior.html]: Could you rename `_renderer` to `_getRenderer`?

File: README.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ Attribute  | Options     | Default              | Description
 `tooltipOptions` | *object* | `{}` | Override/Add Options to your tooltip
 `highchartOptions` | *object* | `{}` | Override/Add Options to the chart initalization code [useful for custom charts]
 `height-responsive` | *Attribute* | `NA` | Make chart height responsive [*define container height for this to work*]
+`renderer` | *object* | `{}` | Allows direct access to the Highcharts rendering layer in order to draw primitive shapes like circles, rectangles,paths or text directly on a chart, or independent from any chart.
```
Comment (avdaredevil) [inline README.md]: Mark in the docs as `[readonly]` like `_chart`

File: highcharts-behavior.html [hunk 3]
```diff
@@ -143,6 +144,7 @@
         _loadingUpdate: function() {if(!this._chart){return};this._chart[(this.loading?"show":"hide")+"Loading"](this.loadingMessage)},
         _subtitleUpdate: function() {if(!this._chart){return};this._chart.setTitle(null,{text: this.subtitle})},
         _hcUpdate: function(o) {if(!this._chart){return};this._chart.update(o)},
+        _renderer: function() {if(!this._chart){return};return this._chart.renderer},
```
Comment (avdaredevil) [inline highcharts-behavior.html]: Rename to `_getRenderer`

**Comment by kevinLB** [comment]
Hi @avdaredevil, I just pushed last modifications about the fixes.
I let you merge when you have time and bump bower version as well.

Thanks for that :+1: 

PS: When I have more time (maybe this weekend), I'll try to offer a "demo" page according to Polymer components. You know with "polymer-cli". It will be based with your Github demo page for sure ;).

Kévin

**Comment by avdaredevil** [comment]
Awesome @kevinLB, I merged your changes, will bump it soon.

Absolutely looking forward to it! Thanks again for your contribution!

**Pull request merged**